index.html

```diff
@@ -1,190 +1,194 @@
-            <button id="copySearch" class="btn btn--secondary btn-sm">Copiar</button>
+          <div class="content-select">
+            <select>
+              <option default>Select language</option>
+              <option>English</option>
+              <option>Spanish</option>
+            </select>
           </div>
         </div>
-        <datalist id="repositories_history">
-          <!-- TODO: ADD repositories history on this datalist -->
-        </datalist>
-        <datalist id="users_history">
-          <!-- TODO: ADD users history on this datalist -->
-        </datalist>
-        
-        
-    </form>
-  </div>
-</div>
-  </main>
-  
-  <script type="module">
-   import { getFromLSOrDefaultValue, validateForm, createSearchFilter, getFormValues } from './src/js/utils.js';
-
-/*
-* STEPS FORM 
-*/
-const btnNext =document.getElementById('btn_next');
-const goBack =document.getElementById('goback');
-const tab1 = document.getElementById('tab_1');
-const tab2 = document.getElementById('tab_2');
-
-btnNext.addEventListener('click',function(evt){
-  evt.preventDefault();
-  tab2.classList.remove('d-none');
-  tab1.classList.add('d-none');
-});
-
-goBack.addEventListener('click',function(evt){
-  evt.preventDefault();
-  tab1.classList.remove('d-none');
-  tab2.classList.add('d-none');
-});
-
-const form = document.forms['calendary_filter'];
-const formData = getFromLSOrDefaultValue('formData', getFormValues(form));
-const linkRedirect = document.getElementById('redirectToSearch');
-const buttonCopy = form['copySearch'];
-
-const fillFormWithFormData = (formData, form) => {
-  if (!form) return;
-
-  for (const [key, data] of Object.entries(formData)) {
-    try {
-      form[key].value = data;
-    } catch (error) {
-      debugger;
-      console.error(error);
-    }
-  }
-}
-
-window.onload = () => {
-  const date = new Date();
-  const fullDate = `${date.getFullYear()}-${date.getMonth()}-${date.getDay() >= 10 ? date.getDate() : '0' + date.getDate() }`;
-
-  form.date.value = fullDate;
-  linkRedirect.classList.add('btn--disabled');
-
-  fillFormWithFormData(formData, form);
-  validateForm(formData, linkRedirect);
-}
-
-form.addEventListener('change', (evt) => {
-  formData[evt.target.name] = evt.target.value;
-  localStorage.setItem('formData', JSON.stringify(formData));
-
-  validateForm(formData, linkRedirect);
-}, true);
-
-buttonCopy.addEventListener('click', (evt) => {
-  evt.preventDefault();
-
-  if (validateForm(formData)) {
-    const {
-      typeOfSearch, date
-    } = createSearchFilter(formData);
-
-    navigator.clipboard.writeText(`${typeOfSearch} ${date}`)
-  }
-})
-
-  </script>
-</body>
+      </div>
+      <div style="display: block">
+        <div class="div">
+          <img src="src/assets/git.png" width="100px" height="100px" />
+        </div>
+        <div>
+          <form
+            id="calendary_filter"
+            style="margin-top: 15px"
```

Comment:
please remove this style and move to class, for example `mt-15`, @Katsu08 please confirm if the tailwind style is correct

Reply:
> please remove this style and move to class, for example `mt-15`, @Katsu08 please confirm if the tailwind style is correct

tailwindcss class doesn't have 15px, just 14px and 16px haha:

Classes | Values
-- | -- 
ml-3.5 | margin-left: 0.875rem; /* 14px */
m-4 | margin: 1rem; /* 16px */